services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -7801,7 +7801,12 @@ public Map<String, Long> getAdminOperationVersionFromControllers(String clusterN
         getSslFactory());
 
     // Get version for standby controllers
-    List<Instance> standbyControllers = getControllersByHelixState(clusterName, HelixState.STANDBY_STATE);
+    List<Instance> standbyControllers = new ArrayList<>();
+    try {
+      standbyControllers = getControllersByHelixState(clusterName, HelixState.STANDBY_STATE);
```

Comment:
Looking at the implementation, if external view is empty, we would end up with the same exception.

Reply:
If external view is empty, we cannot find the leader, then we cannot reach this stage. Am I missing anything?